### DISCUSSION TITLE
iPXE booting with Secure Boot

### DISCUSSION THREAD

**Discussion by maxpain** [siderolabs/talos#9945] [Q&A] [answered]
Hello. Is it possible to generate an `ipxe` script on [factory.talos.dev](https://factory.talos.dev) when Secure Boot is enabled?

**Comment by smira** [comment, marked as answer]
This is not an issue, it's a question, so it should be a discussion. 

It is available & documented https://github.com/siderolabs/image-factory#pxe-frontend-api

But it hasn't been tested much.
